docs/user-guide/timeline.md

```diff
@@ -56,6 +56,13 @@ Sometimes you might be interested to show in the timeline histogram only the tim
 !!! Note
     Map Sync feature need at least GeoServer 2.15.2
 
+## Show hidden layers
+
+When then timeline configuration`showHiddenLayer`is set to *false* *(false by default)*, the layers in timeline gets in sync with time layer's visibility (TOC) i.e when a time layer is hidden or removed, the timeline will not show the respective guide layer. Furthermore, the timeline automatically selects the next available guide layer
+if the **Snap to guide layer** option is enabled in the Timeline settings. If set to *true*, the hidden layer will be shown in the timeline.
+
+<img src="../img/timeline/show-hidden-layer.gif" class="ms-docimage"  style="max-width:700px;" />
+
```

Comment:
These options should be in JSDoc, no need to add a gif to explain this option, so you can remove it from the PR
```suggestion
```

Reply:
Sure